api/AudioContext.json

```diff
@@ -148,6 +129,57 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "options": {
+          "__compat": {
+            "description": "Support for the <code>options</code> parameter.",
```

Comment:
Drop the "Support for the"; we are describing the compat, so for each subfeature we list if it is supported or not, it is redundant. Just plain "<code>options</code> parameter" (no dot either, it isn't a note).